build/Targets/Packages.props

```diff
@@ -3,220 +3,222 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
There was a bunch of strange whitespace. I think your tool did something busted. If you add ?w=1 it'll show the diff without whitespace changes.